.github/workflows/lint.yml

```diff
@@ -32,3 +32,17 @@ jobs:
       run: |
         git diff-index --quiet HEAD -- pwndbg tests
         ./lint.sh
+        
+    - name: Run mypy
+      uses: tsuyoshicho/action-mypy@v4
```

Comment:
Reading the docs, we also want install_types: false and targets: pwndbg

Reply:
I think we also want `filter_mode: nofilter`.

Not sure if that's related to the reason it's not showing results, maybe it's filtered if the actual type error isn't on the modified line.